examples/react/start-appwrite-basic/src/routes/signup.tsx

```
@@ -0,0 +1,73 @@
+import { redirect, createFileRoute } from '@tanstack/react-router'
```

ESLint

[error] Member 'createFileRoute' of the import declaration should be sorted alphabetically.

(sort-imports)